modules/roles_profiles/manifests/roles/win116421h2azure.pp

```diff
@@ -0,0 +1,32 @@
+# Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+class roles_profiles::roles::win116421h2azure {
+  # System
+  include roles_profiles::profiles::disable_services
+  include roles_profiles::profiles::suppress_dialog_boxes
+  include roles_profiles::profiles::files_system_managment
+  include roles_profiles::profiles::firewall
+  include roles_profiles::profiles::network
+  include roles_profiles::profiles::ntp
+  include roles_profiles::profiles::power_management
+  include roles_profiles::profiles::scheduled_tasks
+  ## Is this trying to install each time it's run
+  include roles_profiles::profiles::azure_vm_agent
```

Comment:
I have a fix for that coming in with the win 2012 work. The issues is due to a partial name in hiera.